prime-router/src/main/kotlin/azure/DatabaseSubmissionsAccess.kt

```diff
@@ -48,22 +56,35 @@ class DatabaseSubmissionsAccess(private val db: DatabaseAccess = DatabaseAccess(
 
     /**
      * @param sendingOrg is the Organization Name returned from the Okta JWT Claim.
-     * @param sortOrder sort the table by date in ASC or DESC order.
+     * @param order sort the table in ASC or DESC order.
+     * @param sortColumn sort the table by specific column; default created_at.
      * @param resultsAfterDate is the Action Id of the last result in the previous list.
      * @param limit is an Integer used for setting the number of results per page.
      * @return a list of results matching the SQL Query.
      */
+    /* TODO: (DONE) Needs to take sortColumn */
     override fun <T> fetchActions(
         sendingOrg: String,
         order: SubmissionAccess.SortOrder,
+        sortColumn: SubmissionAccess.SortColumn,
         resultsAfterDate: OffsetDateTime?,
         limit: Int,
         klass: Class<T>
     ): List<T> {
 
+        /* TODO: (DONE) FINALLY, this is where we can apply sortColumn instead of
+         *  defaulting to ACTION.CREATED_AT
+         */
+
+        val column = when (sortColumn) {
+            /* Here is where we can set a column based on sortColumn's enum
+            * value */
+            SubmissionAccess.SortColumn.CREATED_AT -> ACTION.CREATED_AT
+        }
```

Comment:
Note: when you sort-by or query by new column, you might want to sanity check that there's a db index on the field. I'm pretty sure we're good since the cursor uses CREATED_AT

Reply:
note from pairing session: might want to name `column` something more descriptive